apps/dashboard/src/api/billing.ts

```diff
@@ -0,0 +1,8 @@
+import type { GetSubscriptionDto } from '@novu/shared';
+import { get } from './api.client';
+
+export async function getBillingSubscription() {
+  const { data } = await get<{ data: GetSubscriptionDto }>('/billing/subscription');
```

Comment:
Note for a separate PR, we need to add a global error handler for all the data fetching.

Reply:
https://linear.app/novu/issue/NV-4352/dashboard-implement-global-error-handling-with-sentry